.dockerignore

```diff
@@ -1,2 +1,7 @@
-target
-node_modules
\ No newline at end of file
+**/.git
+**/result
+**/*.log
+**/target/
+**/node_modules/
+**/Dockerfile
```

Comment:
why do we ignore Dockerfiles?

Reply:
we do not build dockera in dockers. we build dockers in nix for all dockers from root folder.

Reply:
so if you have child  to build  docker in docker using docker file pleae use nix